index.html

```diff
@@ -31,12 +31,15 @@ <h1>Cree <span class="hide-phone">SRO ⇔ </span>syllabics converter</h1>
     <form>
       <div class="box box-sro">
         <label for="sro"> SRO </label>
-        <textarea id="sro" name="sro" placeholder="tânisi. Type Cree (SRO) here!" autocorrect="off" autocapitalize="off" autofocus lang="cr" autocomplete="off" spellcheck="false"></textarea>
+        <textarea id="sro" name="sro" placeholder="tânisi. Type Cree (SRO) here!" autocorrect="off" autocapitalize="off" autofocus lang="cr" autocomplete="off" spellcheck="false"></textarea>        
         <script>getDefaultTextareaValue('sro')</script>
       </div>
 
       <div class="box box-sro">
-        <label for="syl"> Syllabics </label>
+        <div class="syl-copy-line">
+          <label for="syl"> Syllabics </label>
+          <button class="copy-button">Copy Symbols</button>
```

Comment:
It should say **syllabics**!

```suggestion
          <button class="copy-button">Copy syllabics</button>
```

Reply:
Cool, thank you!  I didn't know the target demographic's familiarity with linguistic terminology, only that my husband would recognize the implication of "symbol" faster than "character" or "syllabic" so it could make the user experience feel less linguisty.  I was thinking the same about SRO, in case you wanted to modify or test that for our demographic; as a linguist myself, in love with orthographies, I didn't know what SRO meant until I read the explanation further down the site.  Strange but true. Context clarifies the current use of both so these are just ideas I'm floating.

Reply:
Culturally, the word "syllabics" means a lot! So it's definitely familiar to the intended audience. Frankly, "symbol" makes it sound somewhat insulting? 

I agree with you about "SRO". This one is less well-known among the community members! I may de-emphasize the term, but I want to make it clear that we're using the _standard_ orthography!